svelte.config.js

```diff
@@ -1,4 +1,4 @@
-import adapter from '@sveltejs/adapter-auto';
+import adapter from '@sveltejs/adapter-node';
```

Comment:
Bin mir hier tatsächlich unsicher, ob wir einen Node-Server benutzen sollen. Lasst das nochmal genauer diskuttieren

Reply:
Was wäre da denn die Alternative?

Reply:
Gibt einige laut Doku (https://kit.svelte.dev/docs/adapters), müssen da mal gucken was wir am ehesten brauchen, sodass wir auch keinen unnötigen Overhead haben

Reply:
Vorteil ist Ansichtssache, zum Einen ist es explizit deklariert und dadurch, dass direkt der richtige Adapter verwendet wird, ist es wohl etwas schneller. Ist aber kein Megavorteil.

Reply:
So wie ich es verstehe installiert der Adapter auf auto relativ viele per default vor, während eine klare Deklaration nur den spezifischen Adapter installiert?

Reply:
> When you create a new SvelteKit project with `npm create svelte@latest`, it installs [`adapter-auto`](https://github.com/sveltejs/kit/tree/master/packages/adapter-auto) by default. This adapter automatically installs and uses the correct adapter for supported environments when you deploy:
> 
> - [`@sveltejs/adapter-cloudflare`](adapter-cloudflare) for [Cloudflare Pages](https://developers.cloudflare.com/pages/)
> - [`@sveltejs/adapter-netlify`](adapter-netlify) for [Netlify](https://netlify.com/)
> - [`@sveltejs/adapter-vercel`](adapter-vercel) for [Vercel](https://vercel.com/)
> - [`svelte-adapter-azure-swa`](https://github.com/geoffrich/svelte-adapter-azure-swa) for [Azure Static Web Apps](https://docs.microsoft.com/en-us/azure/static-web-apps/)
> 
> It's recommended to install the appropriate adapter to your `devDependencies` once you've settled on a target environment, since this will add the adapter to your lockfile and slightly improve install times on CI.

Ist wohl trotzdem empfohlen den richtigen Adapter auszuwählen

Reply:
Ja das habe ich auch gelesen, also bleiben wir bei node?